code/src/App.js

```diff
@@ -1,12 +1,28 @@
+/* eslint-disable max-len */
 import React from 'react';
-import data from './data.json';
+import Artist from 'components/Artist';
+import Album from 'components/Album';
+import 'index.css';
+import AlbumCover from 'components/AlbumCover';
 
-console.log(data);
+import data from './data.json';
 
 export const App = () => {
   return (
     <div>
-      Find me in src/app.js!
+       {data.albums.items.map((album)=> { 
```

Comment:
Looking in the inspector tool it looks like since you're doing a map of albums with the css class "container" that is set to "grid", it sets each album to display grid instead of the whole array. If you put a classname to the parent div that returns the map it should work!